### PR TITLE
Us30 01 permutsearch view

### PR DESCRIPTION
**Pull request by wonecode**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]

File: src/Controller/PermutSearchController.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class PermutSearchController extends AbstractController
+{
+    /**
+     * @Route("permutsearch", name="permutsearch")
```
Comment (LucyMerlier) [inline src/Controller/PermutSearchController.php:13]: honest question : does this work without the first "/" ?

File: templates/permutsearch/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                {% if user.isVisible == true %}
+                    <div class="permut-card mb-5">
+                        <div class="permut-header mb-3 d-flex flex-row align-items-center justify-content-between">
+                            <div class="d-flex flex-row align-items-center text-white">
+                                <img class="rounded-circle me-4" src="https://upload.wikimedia.org/wikipedia/commons/7/7c/Profile_avatar_placeholder_large.png" alt="avatar-user">
+                                <div class="title-card">
+                                    <h3 class="h2 mb-1 text-warning">{{ user.username }}</h3>
+                                    <h4 class="h3">
+                                        {% if user.registeredUser.rome is defined %}
+                                            {{ user.registeredUser.rome }}
```
Comment (LucyMerlier) [inline templates/permutsearch/index.html.twig]: shouldn't this be "user.registeredUser.rome.name"?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Controller/PermutSearchController.php (hunk 1 above)
Comment (wonecode) [reply src/Controller/PermutSearchController.php:13]: yes it does

File: templates/permutsearch/index.html.twig (hunk 2 above)
Comment (wonecode) [reply templates/permutsearch/index.html.twig]: you approved the PR of Mael first, so yes now it should be

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**